index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<title>game</title>
+<head>    
+    <center>
+    <h1>CROSS THE RIVER </h1>
```

Comment:
What h1 is doing here? I told that before as well don't put these kind of tags in head.